Mathlib/RingTheory/Polynomial/Opposites.lean

```diff
@@ -30,7 +31,17 @@ namespace Polynomial
 /-- Ring isomorphism between `R[X]ᵐᵒᵖ` and `Rᵐᵒᵖ[X]` sending each coefficient of a polynomial
 to the corresponding element of the opposite ring. -/
 def opRingEquiv (R : Type*) [Semiring R] : R[X]ᵐᵒᵖ ≃+* Rᵐᵒᵖ[X] :=
-  ((toFinsuppIso R).op.trans AddMonoidAlgebra.opRingEquiv).trans (toFinsuppIso _).symm
+  ((toFinsuppIso R).op.trans <| AddMonoidAlgebra.opRingEquiv.trans {
+    -- TODO: Clean up after https://github.com/leanprover-community/mathlib4/pull/32254
```

Comment:
Happy to merge this, instead of making it a hard dependency. Do you want to tie your name to this? Or can you already in comments provide the term that should go here after the clean-up? That would make cleaning up very easy for anyone to do.